packages/gatsby-source-wordpress/gatsby-config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
We should remove the gatsby-config, now wordpress becomes a theme. We should recommend people installing catch-links instead. Themes have different behavior as plugins.

Reply:
It's been like this for 8+ months, what are the downsides to it being a theme?

Reply:
This is also related to the html fields in processed nodes. Any urls pointing at the origin WP server become relative urls and then catch-links makes those do a client-side route instead of a full refresh. This is very common in WP sites which is why I added it by default

Reply:
Unexpected behaviour when they add their own gatsby-plugin-catch-links, you can end up with different versions... It's better to mimic the behaviour and put it inside the source plugin.

Reply:
Ah, that's interesting and makes sense 🤔 maybe for now lets leave it and I'll make a story in CH to follow up with another PR for that